oidb/pepper/service/src/about.rs

```diff
@@ -0,0 +1,14 @@
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct AboutInfo {
+    git_commit: String,
+}
+
+pub static ABOUT_JSON: Lazy<String> = Lazy::new(|| {
```

Comment:
Can you add comments? What is this?

Reply:
this is for showing some metadata at `/about`.